addon/-private/ember-internals.js

```diff
@@ -1,16 +1,21 @@
 import Ember from 'ember';
 
 let __EMBER_METAL__;
-if (Ember.__loader.registry['@ember/-internals/metal']) {
-  __EMBER_METAL__ = Ember.__loader.require('@ember/-internals/metal');
+let emberMetalPaths = [
+  '@ember/-internals/metal', // ember-source from 3.10
+  '@ember/-internals/metal/index' // ember-source from 3.13
+];
+let metalPath = emberMetalPaths.find(path => Ember.__loader.registry[path]);
```

Comment:
FWIW, we shouldn't use `Array.prototype.find` if we still intend for this library to support IE11 (I'm not sure if we do though?).

Possible future refactor would be to swap this to using `ember-compatibility-helpers` so we don't need iterative looping.

Reply:
Papered this over with `Ember.A` for now.